pkgs/by-name/po/posting/package.nix

```diff
@@ -0,0 +1,65 @@
+{
+  lib,
+  fetchFromGitHub,
+  python3,
+}:
+let
+  py-pkgs = python.pkgs;
+  python = python3.override {
+    packageOverrides = self: super: {
+      # Current nixpkgs version is too outdated, posting maintainer
+      # explicitly warns against relaxing this dependency
```

Comment:
Why is 0.82.0 tagged in October 3th too outdated? What is preventing you from updating it?

Reply:
Thanks for reviewing. The pyproject.toml of posting contains this fragment:
```
"textual[syntax]==0.85.0",
# pinned intentionally
```
I assume this means that older versions of textual will cause bugs. Though I suppose you're right in that nothing is stopping me from updating it. I'll see if I can get that working and update the PR today or tomorrow.